src/components/Autocomplete/components/ComboBox/components/TextField/TextField.tsx

```diff
@@ -1,34 +1,21 @@
 import * as React from 'react';
 
-import {contextTypes} from '../../../types';
+import {ComboBoxContext} from 'components/Autocomplete/components/types';
```

Comment:
This needs to be a relative path. (if you decide to keep it instead of moving the ComboBoxContent int the Context subcomponent)

Reply:
Auto import got me 🙈